schemas/context/webinfo.example.1.json

```diff
@@ -0,0 +1,15 @@
+{
+  "xdm:page": {
```

Comment:
xdm;page is *NOT* a web page.  We've been over this one before...call it xdm;webpage if you wish.